funnel/views/proposal.py

```diff
@@ -71,6 +72,7 @@ def new_proposal(self):
             return redirect(self.obj.url_for(), code=303)
 
         form = ProposalForm(model=Proposal, parent=self.obj)
+        proposal = ''
```

Comment:
This should not mix up `str` and `Proposal` types. Should be `proposal: Proposal = None` to indicate the value is missing.